API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
Should we expect users to read docs for joi.compile for this to make sense?

I know that `var alt = Joi.alternatives().try(Joi.number(), Joi.string())` and `Joi.alternatives([Joi.number(), Joi.string()])`  are equivalent. So I thought that's what you were trying to document here.